.github/workflows/php_versions_tests.yml

```diff
@@ -39,5 +39,5 @@ jobs:
                     php-version: ${{ matrix.php }}
                     coverage: none
 
-            -   run: cd packages/${{ matrix.package }}; composer update --prefer-dist --no-progress
+            -   run: cd packages/${{ matrix.package }}; composer update --prefer-dist --no-progress --ansi
```

Comment:
Colors :rainbow_flag:  ! 

See https://tomasvotruba.com/blog/2020/09/28/how-to-add-colors-to-continuious-integration-output/